### PR TITLE
Disable flow publishing when no actions are present

### PR DESCRIPTION
**Pull request by LisaManresa**
- [x] Add the Clubhouse Story ID: [ch21600]

This prevents a flow from being published if there are no published actions on it.

Only thing left is to sort out styles on the droplist. Supporting a disabled item in the picklist is complicated and requires more discussion.

### CODE REVIEW

File: src/js/views/admin/program/flow/flow_views.js [hunk 1]
```diff
@@ -72,7 +74,12 @@ const HeaderView = View.extend({
     this.showOwner();
   },
   showPublished() {
-    const publishedComponent = new PublishedComponent({ model: this.model, isCompact: true });
+    const publishedComponent = new PublishedComponent({
+      model: this.model,
+      isCompact: true,
+      infoText: intl.admin.program.flowViews.flowStatusInfoText,
```
Comment (paulfalgout) [inline src/js/views/admin/program/flow/flow_views.js]: I'm not 100% sure, but I assumed the infoText would only be there if publish was disabled.  We'll also need to do this same treatment to the flow sidebar

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/views/admin/program/flow/flow_views.js (hunk 1 above)
Comment (LisaManresa) [reply src/js/views/admin/program/flow/flow_views.js]: Good call.

**Comment by LisaManresa** [comment]
Also, just a note on coverage. It's dropped a little since we need to implement the actions logic before we can fully test the publish disabling/enabling. But once the actions are implemented I can quickly write tests to cover the other test cases.

**Comment by LisaManresa** [comment]
[ch21600]

**Review by luketlancaster — APPROVED** [review]
Everything looks good to me, for whatever that's worth 🙃

**Pull request merged**